ui.tests/test-module/libs/commons/localeDataSets.js

```diff
@@ -0,0 +1,227 @@
+const selectors = {
+    ACCORDION: '.cmp-accordion > label',
+    BUTTON: '.button.base:not(.submit):not(.reset) span',
```

Comment:
Please fix the selector, ideally these selectors should be using the BEM style only, since they are publicly documented